package.json

```diff
@@ -13,7 +13,9 @@
     "lint": "eslint . --cache --ext .js,.ts,.tsx,.jsx",
     "ts:lint": "tslint --project .",
     "####### Testing #######": "",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "airtap --local build/src/tests/*.test.js",
```

Comment:
as discussed, `test` should be headless and `test:interactive` should be the browser popping

Reply:
I think `test` should run the unit tests without Airtap, i.e. with Tape on Node. This can be run locally.

`airtap` without the `--local` flag will attempt to spawn out browsers on Sauce Labs. There is no headless process.

Reply:
We discussed live and agreed to leave this as is and implement `test:ci` with `airtap`